torch/utils/collect_env.py

```diff
@@ -89,7 +89,7 @@ def run_and_return_first_line(run_lambda, command):
 
 def get_conda_packages(run_lambda):
     conda = os.environ.get('CONDA_EXE', 'conda')
-    out = run_and_read_all(run_lambda, f"{conda} list")
+    out = run_and_read_all(run_lambda, "{} list".format(conda))
```

Comment:
Maybe add comment that we specifically want older Python compatibility here so this is not updated accidentally.